src/digits/ff31.rs

```diff
@@ -0,0 +1,1373 @@
+///Convert the src into the limbs. This _does not_ mod off the value. This will take the first
+///len bytes and split them into 31 bit limbs.
+///Note that this will _not_ check anything about the length of limbs and could be unsafe... BE CAREFUL!
```

Comment:
What are the failure conditions? Should we add "unsafe" to the name of the function?